package.json

```diff
@@ -41,6 +42,7 @@
   },
   "devDependencies": {
     "concat-stream": "^1.6.2",
+    "eslint": "1.x",
```

Comment:
I would _prefer_ not to "pin" the `eslint` devDependency this way.
`eslint` is currently at version `5.9.0` https://www.npmjs.com/package/eslint
![image](https://user-images.githubusercontent.com/194400/49449469-d98eb000-f7d2-11e8-87c1-3ac7a61cf29c.png)

Setting a _hard_ dependency on `v1.x` is _undesirable_.